packages/core/package.json

```diff
@@ -33,6 +33,7 @@
 	},
 	"dependencies": {
 		"openai": "^4.63.0",
+		"tcp-port-used": "^1.0.2",
```

Comment:
Unsure about this. @msaaddev adding this as a dependency could limit where the framework will work. Like if this `tcp-port-used` pkg isn't supported on Faslty or Workers, etc. I'd like there to not be a dependency like this.

Reply:
Infact i think this is not going to work as this depends on node. We can't add that in core.

Reply:
Ah, makes sense. Let me see what else we can do about it.

Reply:
Removed the dependency.